client/src/lit/curriculum/scrim-inline.js

```diff
@@ -135,6 +159,53 @@ export class ScrimInline extends LitElement {
                   </button>
                 `}
           </div>
+          ${this.survey
+            ? html`<div class="survey">
```

Comment:
Let's extract to `_renderSurvey()` method.

You can even move the condition there:

```js
if (!this.survey) {
  return nothing;
}

// ...
```